src/lib/convertNumberBase.ts

```diff
@@ -0,0 +1,52 @@
+import BigNumber from 'bignumber.js';
+
+import { NumberBaseType } from './numberBaseConvertProperties';
+
+export function convertNumberBase(
+  value: string,
+  fromUnit: NumberBaseType,
+  toUnit: NumberBaseType,
+): string | undefined {
+  for (const [fromBaseName, fromBaseNumber] of Object.entries(
+    numberBaseConversionDict,
+  )) {
+    if (fromBaseName === fromUnit) {
+      for (const [toBaseName, toBaseNumber] of Object.entries(
+        numberBaseConversionDict,
+      )) {
+        if (toBaseName === toUnit) {
+          return convertNumberBaseWithBn(
+            value,
+            toBaseName,
+            fromBaseNumber,
+            toBaseNumber,
```

Comment:
```suggestion
            fromRadix,
            toRadix,
```
1.
MATHEMATICS
the base of a system of numeration.

Reply:
but "the radix or **base** is the number of unique" :c